ui/controlgroup.js

```diff
@@ -0,0 +1,175 @@
+/*!
+ * jQuery UI Controlgroup @VERSION
+ * http://jqueryui.com
+ *
+ * Copyright 2014 jQuery Foundation and other contributors
+ * Released under the MIT license.
+ * http://jquery.org/license
+ *
+ * http://api.jqueryui.com/controlgroup/
+ */
+(function( factory ) {
+	if ( typeof define === "function" && define.amd ) {
+
+		// AMD. Register as an anonymous module.
+		define([
+			"jquery",
+			"./core",
+			"./widget"
+		], factory );
+	} else {
+
+		// Browser globals
+		factory( jQuery );
+	}
+}(function( $ ) {
+
+$.widget( "ui.controlgroup", {
+	version: "@VERSION",
+	defaultElement: "<div>",
+	options: {
+		disabled: null,
+		items: {
+			"button": "input[type=button], input[type=submit], input[type=reset], button, a",
+			"checkboxradio": "input[type='checkbox'], input[type='radio']",
+			"selectmenu": "select"
+		},
+		direction: "horizontal",
+		excludeInvisible: true,
+		classes: {}
+	},
+
+	_create: function() {
+		this._enhance();
+	},
+
+	// To support enhanced option in jQuery mobile we isolate dom manipulation here
```

Comment:
Uppercase "DOM". Also good call on the comment, I was about to complain about the single call above...